src/Sylius/Component/Review/composer.json

```
@@ -64,7 +64,7 @@
     },
     "autoload-dev": {
         "psr-4": {
-            "Sylius\\Component\\Review\\spec\\": "spec/"
+            "Sylius\\Component\\Review\\Tests\\": "Tests/"
```

```suggestion
            "Tests\\Sylius\\Component\\Review\\": "Tests/"
```

Sylius is already using this nomenclature for test sources.